frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/MarkCompletedDialogue.tsx

```diff
@@ -0,0 +1,125 @@
+import { Box, Radio, RadioGroup, Typography } from '@mui/material';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { LegalValueLabel } from 'component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/LegalValueLabel/LegalValueLabel';
+import { useState } from 'react';
+import useFeatureLifecycleApi from 'hooks/api/actions/useFeatureLifecycleApi/useFeatureLifecycleApi';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { SingleVariantOptions } from './SingleVariantOptions';
+
+interface IMarkCompletedDialogueProps {
+    isOpen: boolean;
+    setIsOpen: (open: boolean) => void;
+    onComplete: () => void;
+    projectId: string;
+    featureId: string;
+}
+
+type Status = 'kept' | 'discarded' | 'kept-with-variant';
+
+export const MarkCompletedDialogue = ({
```

Comment:
There's a bunch of state in this component that deserves a test (separate PR is fine)